project-base/storefront/graphql/requests/adverts/queries/AdvertsQuery.graphql

```diff
@@ -1,5 +1,5 @@
-query AdvertsQuery {
-    adverts {
+query AdvertsQuery($categoryUuid: Uuid, $positionName: String) @friendlyUrl {
```

Comment:
Not sure why `@friendlyUrl` was used here. I am not sure if this query can be considered a "friendly URL" one. What do you base it on?

Reply:
Indeed, unnecessary `@friendlyUrl`. Removed.